src/elements/common/fonts.scss

```diff
@@ -2,12 +2,12 @@
     font-weight: normal;
     font-family: 'Lato';
     font-style: normal;
-    src: local('Lato Regular'), local('Lato-Regular'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Regular.woff2') format('woff2'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Regular.woff') format('woff');
+    src: url('https://fonts.googleapis.com/css?family=Lato&display=swap');
```

Comment:
@ksong-box what's the context for this change? will we no longer host the Lato font?

Reply:
Hello! Oh sorry, this was created because of a miscommunication, and I forgot to delete this 😅 . I'll close it rn!